content/21.nutraceuticals.md

```diff
@@ -233,7 +236,8 @@ However, the NICE have provided guidelines for research to investigate the suppl
 In Slovenia, doctors have been advised to provide nursing home patients with vitamin D [@doi:10.1136/bmj.m1820].
 Despite vitamin D deficiency being a widespread issue in the United States [@doi:10.7759/cureus.2741] the National Institutes of Health have stated that there is "insufficient data to recommend either for or against the use of vitamin D for the prevention or treatment of COVID-19" [@url:https://www.covid19treatmentguidelines.nih.gov/adjunctive-therapy/vitamin-d/].
 These are just some examples of how public health guidance has responded to the emerging evidence with regards to vitamin D and COVID-19.
-In contrast, some companies have used the emerging evidence surrounding vitamin D to sell products that claim to prevent and treat COVID-19, which in one incident required a federal court to intervene and issue an injunction barring the sale of vitamin-D-related products, due to a lack of clinical data to support their claims [@url:https://www.justice.gov/opa/pr/court-orders-georgia-defendants-stop-selling-vitamin-d-products-treatments-covid-19-and-other]
+There is also evidence that individuals are paying attention to their seasonal vitamin D levels, as a survey of polish consumers showed that 56% of respondents used vitamin D during the pandemic [@doi:10.3390/nu13010054].  
+However, some companies have used the emerging evidence surrounding vitamin D to sell products that claim to prevent and treat COVID-19, which in one incident required a federal court to intervene and issue an injunction barring the sale of vitamin-D-related products, due to a lack of clinical data to support their claims [@url:https://www.justice.gov/opa/pr/court-orders-georgia-defendants-stop-selling-vitamin-d-products-treatments-covid-19-and-other]
```

Comment:
```suggestion
However, some companies have used the emerging evidence surrounding vitamin D to sell products that claim to prevent and treat COVID-19, which in one incident required a federal court to intervene and issue an injunction barring the sale of vitamin-D-related products due to the lack of clinical data supporting these claims [@url:https://www.justice.gov/opa/pr/court-orders-georgia-defendants-stop-selling-vitamin-d-products-treatments-covid-19-and-other]
```